elrond-wasm-node/src/api/send_api_node_managed.rs

```diff
@@ -0,0 +1,500 @@
+        gas: u64,
+        to: &ManagedAddress<Self::ProxyTypeManager>,
+        amount: &BigUint<Self::ProxyTypeManager>,
+        endpoint_name: &ManagedBuffer<Self::ProxyTypeManager>,
+        arg_buffer: &ManagedArgBuffer<Self::ProxyTypeManager>,
+    ) -> ManagedVec<Self::ProxyTypeManager, ManagedBuffer<Self::ProxyTypeManager>> {
+        unsafe {
+            let result_handle = mBufferNew();
+
+            let _ = managedExecuteOnDestContextByCaller(
+                gas as i64,
+                to.get_raw_handle(),
+                amount.get_raw_handle(),
+                endpoint_name.get_raw_handle(),
+                arg_buffer.get_raw_handle(),
+                result_handle,
+            );
+
+            ManagedVec::from_raw_handle(self.clone(), result_handle)
+        }
+    }
+
+    fn execute_on_same_context_raw(
+        &self,
+        gas: u64,
+        to: &ManagedAddress<Self::ProxyTypeManager>,
+        amount: &BigUint<Self::ProxyTypeManager>,
+        endpoint_name: &ManagedBuffer<Self::ProxyTypeManager>,
+        arg_buffer: &ManagedArgBuffer<Self::ProxyTypeManager>,
+    ) {
+        unsafe {
+            let result_handle = mBufferNew();
+
+            let _ = managedExecuteOnSameContext(
+                gas as i64,
+                to.get_raw_handle(),
+                amount.get_raw_handle(),
+                endpoint_name.get_raw_handle(),
+                arg_buffer.get_raw_handle(),
+                result_handle,
+            );
+
+            // TODO: return result?
+        }
+    }
+
+    fn execute_on_dest_context_readonly_raw(
+        &self,
+        gas: u64,
+        to: &ManagedAddress<Self::ProxyTypeManager>,
+        endpoint_name: &ManagedBuffer<Self::ProxyTypeManager>,
+        arg_buffer: &ManagedArgBuffer<Self::ProxyTypeManager>,
+    ) -> ManagedVec<Self::ProxyTypeManager, ManagedBuffer<Self::ProxyTypeManager>> {
+        unsafe {
+            let result_handle = mBufferNew();
+
+            let _ = managedExecuteReadOnly(
+                gas as i64,
+                to.get_raw_handle(),
+                endpoint_name.get_raw_handle(),
+                arg_buffer.get_raw_handle(),
+                result_handle,
+            );
+
+            ManagedVec::from_raw_handle(self.clone(), result_handle)
+        }
+    }
+
+    fn storage_store_tx_hash_key(&self, data: &ManagedBuffer<Self::ProxyTypeManager>) {
+        let tx_hash = self.get_tx_hash_managed();
+        self.storage_store_managed_buffer_raw(tx_hash.get_raw_handle(), data.get_raw_handle());
+    }
+
+    fn storage_load_tx_hash_key(&self) -> ManagedBuffer<Self::ProxyTypeManager> {
+        let tx_hash = self.get_tx_hash_managed();
+        ManagedBuffer::from_raw_handle(
+            self.clone(),
+            self.storage_load_managed_buffer_raw(tx_hash.get_raw_handle()),
+        )
+    }
+
+    fn call_local_esdt_built_in_function(
+        &self,
+        gas: u64,
+        endpoint_name: &ManagedBuffer<Self::ProxyTypeManager>,
```

Comment:
Same comment as in the other file, this isn't an endpoint, but rather an account-level built-in function.

Reply:
Next PR, I also need to ad tests